components/raftstore/src/store/fsm/peer.rs

```diff
@@ -914,6 +915,53 @@ where
         syncer.report_for_self(self_report);
     }
 
+    // set a flag to stop schedule and rw
```

Comment:
Maybe we can describe the process in more detail to tell why we need to stop the scheduling and RW and wait for the latest index applied.

Reply:
Done.